BlazarUI/app/scripts/components/shared/SearchFilter.jsx

```diff
@@ -19,7 +21,7 @@ class SearchFilter extends Component {
       this.props.onChange(this.refs.searchFilterInput.value);
     }, 250);
   }
-    
+
   componentDidMount() {
     window.addEventListener('keyup', this.handleKeyup);
```

Comment:
it looks like this functionality used to be in the UI but was incompletely removed:
https://github.com/HubSpot/Blazar/commit/793d52746bf8e107b6385586007ee14dde840b93